.github/workflows/docker.yml

```diff
@@ -0,0 +1,48 @@
+name: Build & Push TM-DB-Testing
+on:
+  pull_request:
+    paths:
+      - "tools/*"
```

Comment:
Is this right?

Reply:
yup. I had to double-check myself because of the beautiful yaml/yml syntax.